src/common/inc/CaPTkGUIUtils.h

```diff
@@ -311,10 +311,25 @@ inline std::string getCaPTkDataDir()
   return captk_dataDir;
 }
 
-//! opens the link using Qt's desktop services
+//! opens the link using Qt's desktop services 
 inline bool openLink(const std::string &link)
 {
-  return QDesktopServices::openUrl(QUrl(link.c_str()));
+#ifndef _WIN32
```

Comment:
This is probably needed for macOS, either.

Reply:
After reading more into this, this solution doesn't actually work for Mac anyway -- since 10.11, if a setting called System Integrity Protection is on (which it will be most of the time), LD_LIBRARY_PATH and DYLD_LIBRARY_PATH don't get passed to child processes anyway. I'll switch this to be Linux only.